pandas/core/indexes/base.py

```diff
@@ -2126,13 +2126,19 @@ def fillna(self, value=None, downcast=None):
             Scalar value to use to fill holes (e.g. 0).
             This value cannot be a list-likes.
         downcast : dict, default is None
-            a dict of item->dtype of what to downcast if possible,
+            A dict of item->dtype of what to downcast if possible,
             or the string 'infer' which will try to downcast to an appropriate
             equal type (e.g. float64 to int64 if possible).
 
         Returns
         -------
-        filled : Index
+        filled
```

Comment:
```suggestion
        Index
```

Can just specify return type; don't need named variable in Returns section